libkmod/libkmod-elf.c

```diff
@@ -501,11 +503,15 @@ int kmod_elf_get_modversions(const struct kmod_elf *elf, struct kmod_modversion
 
 	for (i = 0; i < count; i++, off += MODVERSION_SEC_SIZE) {
 		const char *symbol = elf_get_mem(elf, off + crclen);
+		size_t nlen = strnlen(symbol, namlen);
+
+		if (nlen == namlen)
```

Comment:
Nit: let's add a `ELFDBG` for all instances, not just the `get_dependency_symbols` one.

Reply:
Added

Reply:
Can we get one for the `kmod_elf_crc_find()` case as well?

Reply:
That's what I get when I just look for `nlen == namlen`, hah. Adjusted